app/mailers/application_mailer.rb

```diff
@@ -12,7 +12,7 @@ class ApplicationMailer < ActionMailer::Base
 
   default(
     from: -> { email_from },
-    template_path: ->(mailer) { "mailers/#{mailer.class.name.underscore}" },
+    template_path: ->(mailer) { "mailers/#{mailer.polymorphic_type_name.underscore}" },
```

Comment:
is ActionMailer::Base an application record? Or, does this method exist?

Reply:
Looks like my `sed` was overly eager.  I'll patch.